src/Gui/Application.cpp

```diff
@@ -2295,6 +2295,11 @@ void Application::runApplication()
     int argc = App::Application::GetARGC();
     GUISingleApplication mainApp(argc, App::Application::GetARGV());
 
+    // If QT is running with native Wayland then inform Coin to use EGL
```

Comment:
```suggestion
#if defined(FC_OS_LINUX)
    // If QT is running with native Wayland then inform Coin to use EGL
```

Reply:
[TIL!](https://docs.freebsd.org/en/books/handbook/wayland/)

There are other parts of FreeCAD that make the assumption that Wayland is only on Linux.  This is likely a common oversight.